lib/util/twilio-video-errors.js

```diff
@@ -587,7 +587,7 @@ Object.defineProperty(TwilioErrorByCode, 53202, { value: ParticipantIdentityChar
  */
 class ParticipantMaxTracksExceededError extends TwilioError {
   constructor() {
-    super(53203, 'Participant has too many Tracks');
+    super(53203, 'Published Tracks at maximum capacity');
```

Comment:
Can you mention this error somewhere in the blueprint. Probably best under Functional Requirements: https://docs.google.com/document/d/1kaHlFTD4XfdpqcqA5QgMbX_EF83I0eANw-6wKF7i1_4/edit

Reply:
Is there a corresponding error code PR that adds this as a cause?

Reply:
@idelgado It's already added in the blueprint [here](https://docs.google.com/document/d/1kaHlFTD4XfdpqcqA5QgMbX_EF83I0eANw-6wKF7i1_4/edit#heading=h.a4zxg4wb8s92). I will open an error code pr.